components/lib/crash/src/main/java/mozilla/components/lib/crash/service/GleanService.kt

```diff
@@ -0,0 +1,21 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.lib.crash.service
+
+import mozilla.components.lib.crash.Crash
+import mozilla.components.lib.crash.GleanMetrics.CrashMetrics
+
+/**
+ * A [CrashReporterService] implementation for recording metrics with Glean
+ */
+class GleanService : CrashReporterService {
```

Comment:
This is lacking a doc entry in the [README FILE](https://github.com/mozilla-mobile/android-components/blob/5254d1a9d26210cef568963c03e99034ad9946f9/components/lib/crash/README.md)